### PR TITLE
Updated dependencies to get rid of pulsar-io/jdbc related CVE-2020-13692

### PR DESCRIPTION
**Pull request by dlg99**
### Motivation

`mvn clean install verify -Powasp-dependency-check -DskipTests` found various CVEs

### Modifications

Updated dependencies to get rid of pulsar-io/jdbc related CVE-2020-13692

Also upgraded clickhouse lib and suppressed wrongly detected clickhouse
CVEs (client lib matched to server CVEs)

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is already covered by existing tests

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): *YES*
  - The public API: (yes / no)
  - The schema: (yes / no / don't know)
  - The default values of configurations: (yes / no)
  - The wire protocol: (yes / no)
  - The rest endpoints: (yes / no)
  - The admin cli options: (yes / no)
  - Anything that affects deployment: (yes / no / don't know)

### Documentation

Check the box below or label this PR directly (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

**Review by lhotari — CHANGES_REQUESTED** [review]

File: src/owasp-dependency-check-suppressions.xml [hunk 1]
```diff
@@ -41,4 +41,13 @@
...
     <vulnerabilityName regex="true">.*</vulnerabilityName>
   </suppress>
+
+  <!-- clickhouse: security scan matches client lib to the server CVEs -->
+<suppress>
+   <notes><![CDATA[
+   file name: clickhouse-jdbc-0.3.2.jar
+   ]]></notes>
+   <packageUrl regex="true">^pkg:maven/ru\.yandex\.clickhouse/clickhouse\-jdbc@.*$</packageUrl>
+   <cpe>cpe:/a:yandex:clickhouse</cpe>
```
[the remark this answers is not in the document]
Comment (mattisonchao) [reply src/owasp-dependency-check-suppressions.xml:51]: +1

[the remark this answers is not in the document]
Comment (dlg99) [reply src/owasp-dependency-check-suppressions.xml]: @lhotari @mattisonchao 
Fixed. Please take a look.

**Review by mattisonchao — APPROVED** [review]
LGTM +1

**Review by lhotari — APPROVED** [review]
LGTM

**Pull request merged**